app/recipe/views.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import viewsets
+
+from recipe.models import Ingredient, Recipe
+from recipe import serializers
+
+
+class RecipeViewSet(viewsets.ModelViewSet):
```

Comment:
A bit of related reading:
https://travelperk.blogin.co/posts/backend-api-restructuring-and-localization-178909